src/direct/DirectWindow.vala

```diff
@@ -80,6 +80,8 @@ public class DirectWindow : AppWindow {
     public void update_title (File file, bool modified) {
         title = "%s%s (%s) - %s".printf ((modified) ? "*" : "", file.get_basename (),
                                          get_display_pathname (file.get_parent ()), _ (Resources.APP_TITLE));
+        header.title = "%s%s (%s) - %s".printf ((modified) ? "*" : "", file.get_basename (),
+                                                get_display_pathname (file.get_parent ()), _ (Resources.APP_TITLE));
```

Comment:
Seems this would work just as well :joy: 

```suggestion
        header.title = title;
```

Reply:
Yeah this makes sense to me